server/src/main/java/io/druid/segment/loading/LocalDataSegmentKiller.java

```diff
@@ -36,49 +37,33 @@
   public void kill(DataSegment segment) throws SegmentLoadingException
   {
     final File path = getDirectory(segment);
-    log.info("segment[%s] maps to path[%s]", segment.getIdentifier(), path);
+    try {
```

Comment:
can we add a log line that we are killing a segment located at location <somethign>